layouts/authors/list.html

```diff
@@ -37,7 +37,7 @@ <h1 class="text-xl font-bold text-sky-900 my-2 md:my-4 ">Bio</h1>
                 {{ range .Params.social }}
                 <li><a href="{{ .link | absURL }}" target="_blank"
                         class="flex w-16 h-16 items-center  rounded-lg bg-sky-900 text-white">
-                        <i class="fab {{ .icon }} text-2xl mx-auto"></i>
+                        <i class="fa-brands {{ .icon }} text-2xl mx-auto"></i>
```

Comment:
Although the author pages weren't affected, I thought it would be best to use fa-brands here as well.